doc/dynamic-environments/git-environments.mkd

```diff
@@ -45,6 +45,10 @@ future r10k runs. Finally, if there are directories that do not match existing
 branches, r10k will assume that the branches for those environments were delete
 and will remove those environments.
 
+Each repository that r10k needs to communicate with will require a [Deploy Key]
```

Comment:
+1 to adding a brief section on different meow deploy key terms for different providers meow.